sigstore-tuf-root.go

```diff
@@ -0,0 +1,3 @@
+package main
+
```

Comment:
It could be like
const SigstoreTUFRootJSON = `{"signatures":.....}`
without all the escaped  \" sequences